TransferMethod/Sources/View/Widget/AbstractWidget.swift

```diff
@@ -20,11 +20,15 @@
 import Common
 #endif
 import HyperwalletSDK
+import Insights
 import UIKit
 
 /// Represents the abstract widget input
 class AbstractWidget: UIStackView, UITextFieldDelegate {
     var field: HyperwalletField!
+    var pageName: String!
+    var pageGroup: String!
```

Comment:
you do not need to have ! after String. SO remove it

Reply:
Without this there is an error in 
```
required init(coder: NSCoder) {
        super.init(coder: coder)
    }
```
`Property 'self.pageName' not initialized at super.init call`
Workarounds are to either make those variables `optional` or add `!` - what do you suggest?